changelogs/unreleased/5469-composer-description.yml

```diff
@@ -0,0 +1,4 @@
+description: "Add description to the order api request and extract the call from the Canvas component"
+issue-nr: 5469
+change-type: patch
+destination-branches: [master, iso7]
```

Comment:
missing section

Reply:
I removed description from that as that doesn't influence end-user

Reply:
It doesn't always need to affect the end user to be part of the changelog. The lack of a description was visible in the order view.